frontend/src/features/properties/map/acquisition/add/AddAcquisitionFileYupSchema.ts

```diff
@@ -7,4 +7,14 @@ export const AddAcquisitionFileYupSchema = Yup.object().shape({
     .max(500, 'Acquisition file name must be at most ${max} characters'),
   acquisitionType: Yup.string().required('Acquisition type is required'),
   region: Yup.string().required('Ministry region is required'),
+  team: Yup.array().of(
+    Yup.object().shape({
+      // contact: Yup.object().shape({
```

Comment:
commented out code? can this be removed?

Reply:
fixed, it is required.